recipes/pysam/meta.yaml

```diff
@@ -1,26 +1,25 @@
 package:
     name: pysam
-    version: 0.10.0
+    version: 0.11.0
 
 source:
-    fn: pysam-0.10.0.tar.gz
-    url: https://pypi.python.org/packages/87/8e/d2d8238558970df37c7aa01ddec63057a98042334e939b4c1c69cb9a2504/pysam-0.10.0.tar.gz
-    md5: a1f3333ce60f8de542624cb07c792fc0
-    patches:  # [osx]
+  git_url: https://github.com/pysam-developers/pysam.git
+  git_rev: 697c9918338a638d251831d05d503af24c1773d3
```

Comment:
@kyleabeauchamp you could also use https://github.com/pysam-developers/pysam/archive/697c9918338a638d251831d05d503af24c1773d3.tar.gz and don't have the problems with the linter.